app/src/main/java/com/keylesspalace/tusky/components/conversation/ConversationsViewModel.kt

```diff
@@ -67,6 +71,8 @@ class ConversationsViewModel  @Inject constructor(
                     lastStatus = conversation.lastStatus.copy(expanded = expanded)
             )
             database.conversationDao().insert(newConversation)
+                    .subscribe()
```

Comment:
I don't think it makes much sense to add them to disposables in all 3 cases. We don't want to cancel these actions when the screen is closed, right? We still want to update DB or something.